deployments/gcp-uscentral1b/Makefile

```diff
@@ -0,0 +1,71 @@
+PROJECT?=pangeo-181919
+NAMESPACE?=staging
+ZONE?=us-central1-b
+CLUSTER?=pangeo-uscentral1b
+
+# Creation of the Kubernetes Cluster for Google
+#
+# Notes
+# -----
+# The pangeo-rbac stuff is untested.
+#
+# References
+# ----------
+# https://cloud.google.com/kubernetes-engine/docs/how-to/node-auto-provisioning
+# https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+
+# ----------------------------------------------------------------------------
+# Kubernetes Cluster
+#
+#
+# TODO: enable nodepool auto-provisioning
+		# --enable-autoprovisioning --min-cpu 1 --min-memory 1 --max-cpu 1000 --max-memory 5200 \
+		# --autoprovisioning-service-account=pangeo \
+
+test:
+	@echo $(CLUSTER) $(WORKER_POOL)
+
+up:
+	echo "[Creating cluster pool in $(ZONE)]"
+	gcloud container clusters create $(CLUSTER) \
+		--num-nodes=1 \
+		--zone=$(ZONE) \
+		--cluster-version=latest \
+		--no-enable-ip-alias \
+        --no-enable-legacy-authorization \
+        --enable-autoupgrade --enable-autorepair --max-surge-upgrade=1 \
+		--machine-type=n1-standard-1 \
+		--enable-autoscaling --min-nodes=1 --max-nodes=2 \
+        --node-labels="hub.jupyter.org/nodepurpose=core"
+		--enable-autoprovisioning --autoprovisioning-config-file autoprovisioning.json \
+		--autoprovisioning-service-account=pangeo \
+		--enable-vertical-pod-autoscaling \
+		--workload-metadata=GKE_METADATA \
+		--workload-pool=$(PROJECT).svc.id.goog \
+        --enable-stackdriver-kubernetes
+	gcloud container clusters get-credentials $(CLUSTER) --zone=$(ZONE)
+	# kubectl create namespace $(NAMESPACE)
+	# kubectl -n $(NAMESPACE) apply -f ../../pangeo-deploy/templates/pangeo-rbac.yaml
```

Comment:
@TomAugspurger - this line was failing for me so I just commented out the rest for now.

Reply:
Sounds good. Do you have the error message handy? This is one thing I didn't try earlier. I had my own pangeo-rbac.yaml file.